ethsigner/app/src/test/java/tech/pegasys/ethsigner/app/FileBasedSignerCliConfigTest.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.app;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import tech.pegasys.ethsigner.FileBasedSignerCliConfig;
+
+import org.apache.logging.log4j.Level;
+import org.junit.Test;
+import picocli.CommandLine;
+
+public class FileBasedSignerCliConfigTest {
+
+  private static final String THIS_IS_THE_PATH_TO_THE_FILE = "/this/is/the/path/to/the/file";
```

Comment:
What is the file? ...wouldn't `PASSWORD_FILE` be a better fit? (with a separate file for the key file location)

Reply:
It might be a bit too long, but I still like it, because that is what it is. I've fixed it!